service/core/api/src/main/scala/com/lightbend/lagom/internal/api/transport/LagomServiceApiBridge.scala

```diff
@@ -115,6 +115,7 @@ trait LagomServiceApiBridge {
 
   // Exceptions
   def newPayloadTooLarge(msg: String): Throwable
+  def newBadRequest(msg: String): Throwable
```

Comment:
Yeah, this method is only needed if we need a `*dsl` specific way of creating a bad request from our generic server/client implementations, which we don't.